client/state/purchases/selectors/should-revert-atomic-site-before-deactivation.js

```diff
@@ -1,10 +1,14 @@
 import { isAtomicSupportedProduct } from '@automattic/calypso-products';
+import { isMarketplaceProduct } from 'calypso/state/products-list/selectors';
 import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
 import { getByPurchaseId } from './get-by-purchase-id';
 import { getSitePurchases } from './get-site-purchases';
 
 import 'calypso/state/purchases/init';
 
+const isAtomicSupported = ( productSlug ) =>
+	isAtomicSupportedProduct( productSlug ) || isMarketplaceProduct( productSlug );
```

Comment:
I think that `isMarketplaceProduct` requires two arguments?

Reply:
Absolutely, thanks @sirbrillig for pointing this out! 🎉